app/build.gradle.kts

```diff
@@ -71,6 +72,16 @@ android {
             excludes.add("/META-INF/{AL2.0,LGPL2.1}")
         }
     }
+    firebaseTestLab {
```

Comment:
Is it worth making this block conditional on the file being present? or is it safe to include even when file is missing?

Reply:
I think it's safe, the file is only needed when we run the Gradle task for FTL ./gradlew app:myFtlDeviceDemoDebugAndroidTest. If the file is not found it will give error ```> Unable to find the application-default credentials to send a request to Firebase TestLab.```. 

If developers want to test locally they can add their service-account json to the app codebase path ./app/service-account.json.